packages/loader/driver-utils/src/network.ts

```diff
@@ -46,14 +46,16 @@ export class GenericNetworkError extends LoggingError implements IDriverErrorBas
 export class AuthorizationError extends LoggingError implements IAuthorizationError {
     readonly errorType = DriverErrorType.authorizationError;
     readonly canRetry = false;
+    readonly claims: { tag: string, value: string | undefined }; // See ITaggedTelemetryPropertyType
```

Comment:
why tag: string, should it be tag: TelemetryDataTag

Reply:
Good catch. Although since then, we decided that we'd go with `tag: string` so I'm leaving it.